.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.6.6
```

Comment:
Any reason not to go with 3.1.0 like everything else?

Reply:
It originally used Ruby 2, so I figured some dependencies might not work. I'm happy to upgrade. One sec

Reply:
Wonderful.